src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -257,48 +292,72 @@ private void attachCaseByCcdId(
         Long exceptionRecordId,
         String idamToken,
         String userId,
-        CaseDetails exceptionRecord
+        CaseDetails exceptionRecordData,
+        boolean ignoreWarnings
     ) {
         log.info("Attaching exception record '{}' to a case by CCD ID '{}'", exceptionRecordId, targetCaseCcdRef);
 
-        CaseDetails theCase = ccdApi.getCase(targetCaseCcdRef, exceptionRecordJurisdiction);
-        List<Map<String, Object>> targetCaseDocuments = getScannedDocuments(theCase);
-
-        verifyExceptionRecordIsNotAttachedToCase(exceptionRecordJurisdiction, exceptionRecordId);
-
-        //This is done so exception record does not change state if there is a document error
-        checkForDuplicatesOrElse(
-            exceptionRecordDocuments,
-            targetCaseDocuments,
-            ids -> throwDuplicateError(targetCaseCcdRef, ids)
-        );
-
-        List<Map<String, Object>> newCaseDocuments = attachExceptionRecordReference(
-            exceptionRecordDocuments,
-            exceptionRecordId
-        );
+        Validation<String, Classification> classificationValidation =
+            hasJourneyClassificationForAttachToCase(exceptionRecordData);
```

Comment:
We've already got a validation stage in this service. We need to include classification validation there and hot here, so that it's all in one place and the caseworker receives all errors at once (which is the expected behaviour). It's also not ideal to have validation spread across the processing code.

Same with exception record validation.